Pipfile

```diff
@@ -20,7 +20,7 @@ Django = "==3.1.3"
 celery = "*"
 langdetect = "*"
 django-elasticsearch-dsl = "*"
-textract = "*"
+textract = "==1.5.0"
```

Comment:
Why are we fixing at this exact version?

Reply:
because previous version's dependency required six in version 12 exactly but some other dependency required six in version 14+. this version doesn't require exact six version.